### PR TITLE
Add more free subdomain providers

### PR DESCRIPTION
**Pull request by asheeeshh**
Added 2 new subdomain providers ~ ~`js.cool`~ and `runs-on.tech`, and shifted `is-a.dev` subdomain from other resources to subdomain list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1045,6 +1045,9 @@ Table of Contents
   * [js.org](https://js.org) - Free js.org subdomains for GitHub Pages for the JavaScript community.
   * [pp.ua](https://nic.ua/) — Free pp.ua subdomains.
   * [thedev.id](https://thedev.id) — A free thedev.id subdomain for developers.
+  * [js.cool](https://js.cool) - Free js.cool subdomain for developers.
```
Comment (ripienaar) [inline README.md:1048]: This has a requirement `Your site must have a Chinese translation edition` which I would say disqualifies it.

**Review by ripienaar — CHANGES_REQUESTED** [review]
Thanks, lets remove that one and it's good to go

File: README.md (hunk 1 above)
Comment (asheeeshh) [reply README.md:1048]: alright

**Comment by asheeeshh** [comment]
will try to keep finding some more as domains are something most of the devs are in need of.

**Comment by ripienaar** [comment]
thank you

**Pull request merged**